webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Overview/CardsRegistry.js

```diff
@@ -2,11 +2,17 @@ import React from 'react';
 import { addGlobalFill } from '../../../common/Fill/GlobalFill';
 import AuditCard from '../../Audits';
 import DetailsCard from '../../DetailsCard';
+import RecentCommunicationCard from '../../RecentCommunicationCard';
 import AggregateStatus from '../../Status/AggregateStatusCard';
 
 const cards = [
-  { key: '[core]-detail-card', Component: DetailsCard, weight: 4000 },
-  { key: '[core]-status-card', Component: AggregateStatus, weight: 3500 },
+  { key: '[core]-status-card', Component: AggregateStatus, weight: 4000 },
+  {
+    key: '[core]-recent-comms-card',
```

Comment:
can you spell this differently or add it to the ignore file?

Reply:
sure, where would I add it?

Reply:
`.eslintrc`

Reply:
this = `comms`